spec/unit/enterprise_spec.rb

```diff
@@ -0,0 +1,64 @@
+describe Travis::Enterprise, set_app: true do
+    let(:license_url) { "https://fake.fakeserver.com:9880/license/v1/license" }
+    let(:body) { Travis::Enterprise.check_license_seat? }
+
+    before do
+      Travis.config.enterprise = true
```

Comment:
```suggestion
      Travis.config.stubs(:enterprise).returns(true)
```

Other tests are failing, because enterprise mode is enabled and never disabled

Reply:
Meh, looks like `stubs` doesn't work with Hashr (config is using it under the hood). You can just set it to false in the `after` block